### DISCUSSION TITLE
Does GlobaLeaks remove metadata (e.g. exif info from images)?

### DISCUSSION THREAD

**Discussion by mjakubowskilkr** [globaleaks/globaleaks-whistleblowing-software#4549] [Q&A] [answered]
As in subject.

**Comment by evilaliv3** [comment, marked as answer]
Thank you for your feedbck @mjakubowskilkr 

Actually within the context of GlobaLeaks we have currently intentionally considered to not implement a feature for removing exif info from images for the fact that exif information like any other metadata is considered an important part of the evidence.

For this reason metadata removal is typically considered as an action to be carefully performed by recipients that could discern what is important to preserve and what shall be redacted.

This aspect is detailed in other tickets like https://github.com/globaleaks/GlobaLeaks/issues/305 and documented in many sections of the documentations like https://docs.globaleaks.org/en/main/security/ThreatModel.html#metadata

I invite you to check these other pages and integrate there your recommendations.
